docs/guides/cli/20_proxy-and-tunnel.mdx

```diff
@@ -0,0 +1,376 @@
+      "kid": "f8f2e6ff-0480-4343-9dee-0d2a463146dc",
+      "crv": "P-256",
+      "alg": "ES256",
+      "x": "Say2LSWvHxUnyxuW5lxsTFkKopZq402eH4YqcRiBgvA",
+      "y": "7XaYgYsW-Mjb5qIq47LxyaPHjPZfRHRnnfir8aqd9BU"
+    }
+  ]
+}
+```
+
+You can validate and decode the JWT using the public key.
+
+The token contains all session and identity information linked to the Ory Session
+Cookie. 
+
+This is an example of a JWT with session and identity data: 
+
+```json
+{
+  "exp": 1627560124,
+  "iat": 1627560064,
+  "iss": "https://<project-slug>.projects.oryapis.com",
+  "jti": "eaaec243-2cb0-468a-bebe-1a0349242cf0",
+  "nbf": 1627560064,
+  "session": {
+    "id": "b73d7dc4-f565-4fea-951e-8c23ee05783f",
+    "active": true,
+    "expires_at": "2021-07-30T12:01:02.96663Z",
+    "authenticated_at": "2021-07-29T12:01:03.024365Z",
+    "issued_at": "2021-07-29T12:01:02.966652Z",
+    "identity": {
+      "id": "0f0c9bec-6b68-47f3-b62b-7750f065359c",
+      "schema_id": "default",
+      "schema_url": "https://<project-slug>.projects.oryapis.com/api/kratos/public/schemas/default",
+      "state": "active",
+      "state_changed_at": "2021-07-29T12:01:02.844472Z",
+      "traits": {
+        "email": "dev+docs@ory.sh",
+        "firstname": "Ory Docs",
+        "vegeterian": true
+      },
+      "verifiable_addresses": [
+        {
+          "id": "64b3f270-70fc-48b8-8704-8085c783362b",
+          "value": "dev+docs@ory.sh",
+          "verified": false,
+          "via": "email",
+          "status": "sent",
+          "verified_at": null,
+          "created_at": "2021-07-29T12:01:02.888562Z",
+          "updated_at": "2021-07-29T12:01:02.888562Z"
+        }
+      ],
+      "recovery_addresses": [
+        {
+          "id": "526a2b25-2b9a-445b-982d-97862d9bbc9b",
+          "value": "dev+docs@ory.sh",
+          "via": "email",
+          "created_at": "2021-07-29T12:01:02.897407Z",
+          "updated_at": "2021-07-29T12:01:02.897407Z"
+        }
+      ],
+      "created_at": "2021-07-29T12:01:02.877417Z",
+      "updated_at": "2021-07-29T12:01:02.877417Z"
+    }
+  },
+  "sub": "0f0c9bec-6b68-47f3-b62b-7750f065359c"
+}
+```
+
+
+### Usage in Local Development
+
+When using the Proxy for local development, point to the URL where your application runs. 
+
+For example, if your application is available at `http://localhost:3000`, run this command: 
+
+```shell
+ory proxy http://localhost:3000
+```
+
+By default, the Proxy creates an entry point at `http://localhost:4000`. To get access to Ory endpoints, 
+you must access the app through the proxy using This URL instead of the actual address on which your
+application is running. 
+
+If you want to adjust the entry point URL, pass the desired address along with the application URL, 
+for example: 
+
+```shell
+ory proxy http://localhost:3000 http://localhost:3001
+```
+
+### Usage in Production
+
+You can use Ory Proxy in a production setup when you are self-hosting Ory components. 
```

Comment:
what do you mean by components? 
I thought you just self-host the user interface / frontend? 
I think ory proxy only works with ory cloud.